### ISSUE TITLE
exit - navigation

### ISSUE THREAD

**Issue by oberon-manjaro** [Chrysostomus/bmenu#11]
It would be great to use '0' everywhere in the menu tree to navigate one level up.
At the moment I find it confusing and in some places not logical that for example when you are inside a sub-menu of >Appearance you need a different number depending on how many options are available.
Also in this situation I'd expect 'cancel' to directed me back to 'Appearance' and not two levels up to the main menu.
I know that **mhwd-tui** is actually not the same piece of software, but of course when navigating the options you have the feeling that you stay inside the same application.
Therefor it is confusing that in mhwd-tui exit navigation is different again.
I would expect when I am for example inside the option 'Install kernel', that '0' whould bring me back to mhwd-tui.
Ctrl+c should maybe be the shortcut to abort and exit bmenu altogether, no matter where you currently are.

**Comment by Chrysostomus** [comment]
Yeah, there is much room for improvement here. I agree with your idea. Unfortunately I know this side of bash poorly. Patches are welcome =). But I'll see what I can do about that. Items using select are difficult, because numbers are assigned to items automatically. I use it mainly in places where I don't know number of items beforehand (like choosing kernels).

**Comment by oberon-manjaro** [comment]
You're doing a fantastic job, @Chrysostomus ! I would not even know where to begin... But I will see what I can do if I find some time, soon :)
Thank you!
